lib/utilities.js

```diff
@@ -118,6 +127,11 @@ utilities._getResourcesLinks = function _getResourcesLinks (links) {
         delete link.filter
       }
 
+      if (!_.isEmpty(link.context)) {
+        hrefParameters.context = _.cloneDeep(link.context)
+        delete link.context
```

Comment:
Why are you cloning first and delete after? Does that not make the cloning useless?